bin/reference_data.py

```diff
@@ -152,9 +152,23 @@ def prepare_metadata(self):
                 'variation': os.path.join(self.rel_data_dir, os.path.basename(self.reference_variation))
             }
         })
+
+        if 'faidx' in self.procs:
+            self.procs['faidx'].wait()
+        defaults = {}
+        if os.path.exists(self.reference_fasta + '.fai'):
+            genome_size, chromosome_count = self.genome_param_from_fai_file(self.reference_fasta + '.fai')
+            defaults = {
+                'genome_size': genome_size,
+                'chromosome_count': chromosome_count,
+                'goldenpath': genome_size
```

Comment:
Genome size and GoldenPath are being set to the same value - is this what we want?

Reply:
It being given the same default value but that will be changeable. in the input bellow.
The rational is that not having a default from `goldenpath` will probably result in the value not being entered.